### PR TITLE
Enable fatalwarnings for most Alpakka and internal projects

### PR DESCRIPTION
**Pull request by seglo**
References #2456

### CODE REVIEW

**Comment by seglo** [comment]
Re-evaluate later.

**Comment by seglo** [comment]
Took another crack at this because of #2456. All projects have `fatalWarnings := true` except for OrientDB which needs a refactor.

File: build.sbt [hunk 1]
```diff
@@ -107,56 +107,49 @@ TaskKey[Unit]("verifyCodeFmt") := {
...
 
 lazy val ftp = alpakkaProject(
   "ftp",
   "ftp",
   Dependencies.Ftp,
   Test / fork := true,
   // To avoid potential blocking in machines with low entropy (default is `/dev/random`)
-  javaOptions in Test += "-Djava.security.egd=file:/dev/./urandom",
-  fatalWarnings := true
+  javaOptions in Test += "-Djava.security.egd=file:/dev/./urandom"
```
Comment (marcospereira) [inline build.sbt:152]:
```suggestion
  Test / javaOptions += "-Djava.security.egd=file:/dev/./urandom"
```

Comment (marcospereira) [reply build.sbt:152]: To be sbt 1.5 friendly.

File: influxdb/src/main/scala/akka/stream/alpakka/influxdb/impl/AlpakkaResultMapperHelper.scala [hunk 2]
```diff
@@ -63,7 +63,7 @@ private[impl] class AlpakkaResultMapperHelper {
         val columnName: String = column.name()
         val fieldType: Class[_] = field.getType()
 
-        if (!field.isAccessible()) {
+        if (!field.canAccess(model)) {
```
Comment (marcospereira) [inline influxdb/src/main/scala/akka/stream/alpakka/influxdb/impl/AlpakkaResultMapperHelper.scala:66]: Not sure if this is relevant here, but I don't think `canAccess` exists in Java 8.

Comment (seglo) [reply influxdb/src/main/scala/akka/stream/alpakka/influxdb/impl/AlpakkaResultMapperHelper.scala:66]: Ah, you're right. I'll silence this warning.

File: build.sbt (hunk 1 above)
Comment (seglo) [reply build.sbt:152]: True, but I think we should tackle this later when we upgrade.

**Review by marcospereira — APPROVED** [review]

**Pull request merged**